content/millennium/r4/clinical/medications/immunization.md

```diff
@@ -69,23 +74,33 @@ Search for Immunizations that meet supplied query parameters:
 
  Name             | Required?          | Type          | Description
 ------------------|--------------------|---------------|-----------------------------------------------------------------------------------------------------
- `_id`            | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
- `patient`        | This, or `_id`     | [`reference`] | The patient for the vaccination record. Example: `12345`
- `date`           | No                 | [`date`]      | Date range into which the immunization administration date falls. Must be prefixed by 'ge' or 'le'.
- `_revinclude`    | No                 | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
- `target-disease` | No                 | [`token`]     | The target disease the dose is being administered against.                                 Example: `http://hl7.org/fhir/sid/cvx|213`
-
-Notes:
-
-* The `date` parameter may be provided:
-  * once with a prefix `ge` or `le` representing the earliest date or latest date. (e.g. `date=ge2015-01-01`, `date=le2016-01-01`)
-  * twice with the prefixes `ge`, `le` to indicate a specific range. (e.g. `date=ge2015-01-01&date=le2016-01-01`)
-* The `_revinclude` parameter may be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
-* The `_revinclude` parameter may be provided in combination with the `_id/patient` parameter. Example: `_id=M17255835,M17255827&_revinclude=Provenance:target`
-* When `_revinclude` is provided in a request to a closed endpoint, the OAuth2 token must include the `user/Provenance.read` scope.
-* **Currently the `patient/Provenance.read` scope is not supported and hence `_revinclude` cannot be utilised for patient persona.**
-* The `target-disease` parameter
-  * Searching by Millennium proprietary codes is not supported
+ `_id`            | Conditionally      | [`token`]     | The logical resource ID associated with the resource. This parameter is required if the `patient` parameter is not used. Example: `M12345`
```

Comment:
@cldeitrich, some of these PRs now say "Conditional" and some say "Conditionally". which is the preference for consistency?

Reply:
"Conditionally" matches what we use in one of the Reference Page templates, so that would be my preference.